plugins/Login/Controller.php

```diff
@@ -118,6 +121,19 @@ function login($messageNoAccess = null, $infoMessage = false)
         return $view->render();
     }
 
+    protected function getLoginFromLoginOrEmail($loginOrEmail)
```

Comment:
Be good to have this method ideally outside the controller or make it at least private so it can for sure not be executed?

Reply:
changed the method to private